pkg/controller/operators/catalog/operator.go

```diff
@@ -1387,28 +1387,36 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 
 	querier := o.serviceAccountQuerier.NamespaceQuerier(plan.GetNamespace())
 	ref, err := querier()
-	if err != nil {
-
-		// Retry sync if non-fatal error
-		if !scoped.IsOperatorGroupError(err) {
```

Comment:
We can remove the associated error util here since it's only being used here.
https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/lib/scoped/error.go

But I'm okay with making this a follow up too.

Reply:
It looks harmless + we could end up needing it later on. Feels like it's better to leave that method in there just for the sake of completeness of the type definition.